hooks/post_gen_project.py

```diff
@@ -6,6 +6,11 @@ def git_init():
     subprocess.run(["git", "init", "-b", "main"])
 
 
+def dvc_init():
+    print("Initializing dvc")
+    subprocess.run(["dvc", "init"])
+
```

Comment:
dvc files were not yet added. Can we do this here `subprocess.run(["git", "commit", "-m", "Initialize DVC"])`?